src/components/flash/flash.js

```diff
@@ -305,19 +318,19 @@ class Flash extends React.Component {
           key={ title }
           title={ title }
           open={ this.state.dialogs[title] || false }
-          onCancel={ this.toggleDialog.bind(this, title) }
+          onCancel={ this.toggleDialog(title) }
         >
           { desc }
         </Alert>
       );
 
       // create text for item
-      text = (
+      return (
```

Comment:
Yeah I converted it because it's wrapped inside an `if` statement and the linter raised an error because it didn't like `text` being redefined.

There is still a `return text;` at the bottom for when the if statement is false. This might be clearer if it was inside an `else` but the linter would raise an error on that because there is another rule that says the `else` is not technically required and should be removed.